Sources/Localization/en.lproj/Localizable.strings

```diff
@@ -64,14 +65,18 @@
 // Add Transfer Method
 "create_account_label" = "Create Account";
 "enter_your_information_formatter" = "Enter your %@ information";
-"account_header" = "%1$@ - %2$@ (%3$@)";
 "bank_account_footer" = "On a paper check, the routing number is printed as the first 9 digits on the bottom of the check, followed by 4-17 digits for the account number.";
 "bank_card_footer" = "CVV is a 3 digit number, typically found on the back of the card.";
 "paypal_account_footer" = "PayPal transfer method email address should be same as profile email address.";
-"add_profile_footer" = "Note: we are not able to support adding an account for someone else.";
-"profile_header"="Account Holder";
+"account_holder_footer" = "Note: we are not able to support adding an account for someone else.";
```

Comment:
We have removed footer from the code. Footer for now will have only error message if any. No constant text. So you can remove this footer

Reply:
fixed this one